sdk-installer/test/integration/test_integration_python.py

```diff
@@ -41,25 +41,28 @@ def __init__(self):
         return False
 
 
-def test_no_sdk_reference_found__latest_version_installed():
+def test_no_sdk_reference_found__nothing_installed():
     requirements_contents = """
 
     """
-    with open("./app/requirements-velocitas.txt", mode="w") as requirements:
+    with open(
+        "./app/requirements-velocitas.txt", mode="w", encoding="utf-8"
+    ) as requirements:
         requirements.write(requirements_contents)
 
     subprocess.check_call(["velocitas", "init", "-f", "-v"], stdin=subprocess.PIPE)
-    assert is_package_installed("velocitas_sdk") or is_package_installed(
+    assert not is_package_installed("velocitas_sdk") and not is_package_installed(
         "velocitas-sdk"
     )
-    assert can_import_and_use_vehicleapp()
 
 
 def test_sdk_reference_found__sdk_installed():
     requirements_contents = """
 velocitas_sdk==0.13.0
```

Comment:
can we keep this somehow automatically up-to-date? or what about using latest/main?

Reply:
Not easily doable without an extra http request to figure out the latest tag. At the moment it does not hurt to reference an outdated SDK tag since we just want to check whether it is intalled or not.